sg_otio/sg_cut_track_writer.py

```diff
@@ -797,7 +797,8 @@ def write_shots(self, clips_by_shots, sg_project, sg_linked_entity, sg_user=None
         sg_batch_data = []
         sg_shots = []
         for shot_name, clip_group in clips_by_shots.items():
-            if not shot_name:
+            # The shot name might be _no_shot_name, so we need to check the clip group name too.
+            if not shot_name or not clip_group.name:
```

Comment:
Should we just check `clip_group.name`? Is it possible to have an empty group name but a shot name?